src/Magnum/GL/Buffer.h

```diff
@@ -888,7 +888,7 @@ class MAGNUM_GL_EXPORT Buffer: public AbstractObject {
          * @overload
          * @m_since{2020,06}
          */
-        template<class T> explicit Buffer(TargetHint targetHint, std::initializer_list<T> data, BufferUsage usage = BufferUsage::StaticDraw): Buffer{targetHint, Containers::arrayView(data), usage} {}
+        template<class T> explicit Buffer(TargetHint targetHint, Containers::ArrayView<T> data, BufferUsage usage = BufferUsage::StaticDraw): Buffer{targetHint, Containers::arrayView(data), usage} {}
```

Comment:
This is probably not a change you wanted to do -- it's already delegating to the ArrayView overload above. Same for the `setData()` overloads below.

Reply:
Oh, right, careless use of search and replace hehe, reverted